packages/cxl-lumo-styles/scss/themes/vaadin-context-menu-item.scss

```diff
@@ -8,3 +16,8 @@
 ::slotted(a:not(:hover)) {
   text-decoration: none !important; /* stylelint-disable-line declaration-no-important */
 }
+
+::slotted(a:hover) {
+  color: var(--lumo-primary-color) !important; /* stylelint-disable-line declaration-no-important */
+  text-decoration: none !important; /* stylelint-disable-line declaration-no-important */
```

Comment:
Can this go into `::slotted(a)`? `::slotted(a:not(:hover))` has this rule too, maybe we can just disable it without repeating?